docs/tutorials/02-local-simple-blueprint.md

```diff
@@ -0,0 +1,438 @@
+        baseUrl: eu.gcr.io/gardener-project/landscaper/tutorials/components
+      componentName: github.com/gardener/landscaper/ingress-nginx
+      version: v0.2.0
+```
+
+__imports__:
+The blueprint needs a target import of type kubernetes cluster.
+The target `mycluster` is created as mentioned above and has to be connected to the import.
+This is done by specifying the target as targets import.
+
+:warning: The "#" has to be used to reference the previously created target. Otherwise, the landscaper would try to import the target from another component's export.
+```yaml
+imports:
+  targets:
+  - name: cluster
+    # the "#" forces the landscaper to use the target with the name "my-cluster" in the same namespace
+    target: "#my-cluster"
+```
+
+__exports__:
+
+The nginx ingress blueprint export the used ingressClass so that it can be reused by other components.
+To give the generic ingress class more semantic meaning in the current installation, the export is exported as `myIngressClass`.
+Other installation are now able to consume the data with this specific name.
+
+:warning: Note that this name has to be unique so that it will not be overwritten by other installations.
+
+The export is a data object export, therefore the eyport is defined under `spec.exports.data` and is written to the `dataRef: myIngressClass`.
+```yaml
+exports:
+  data:
+  - name: ingressClass
+    dataRef: "myIngressClass"
+```
+
+```yaml
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Installation
+metadata:
+  name: my-ingress
+spec:
+  blueprint:
+    ref:
+      repositoryContext:
+        type: ociRegistry
+        baseUrl: eu.gcr.io/gardener-project/landscaper/tutorials/components
+      componentName: github.com/gardener/landscaper/ingress-nginx
+      version: v0.2.0
+      resourceName: ingress-nginx-blueprint
+
+  imports:
+    targets:
+    - name: cluster
+      # the "#" forces the landscaper to use the target with the name "my-cluster" in the same namespace
+      target: "#my-cluster"
+  
+  exports:
+    data:
+    - name: ingressClass
+      dataRef: "myIngressClass"
+```
+
+When the `Target` and the Installation CRs are properly configured, they can be applied to the kubernetes cluster running the landscaper.
+
+```shell script
+kubectl create -f docs/tutorials/resources/ingress-nginx/my-target.yaml
+kubectl create -f docs/tutorials/resources/ingress-nginx/installation.yaml
+```
+
+The landscaper will then immediately start to reconcile the installation as all imports are satisfied.
+
+The first resources that will be created is the execution object which is a helper resource that contains the rendered deployitems.
+The status shows the one specified Helm deploy item which has been automatically created by the landscaper.
+```shell script
+$ kubectl get inst
+NAME                        PHASE       CONFIGGEN   EXECUTION                   AGE
+my-ingress                  Succeeded               my-ingress                  4m11s
```

Comment:
does the `installation` move to phase `succeeded` as soon as the `execution` has been created?

Reply:
if everything is correct everything should succeed in seconds. But I will update it to be progressing which seems more redable in the tutorial.